lib/Controller/PageController.php

```diff
@@ -69,7 +69,7 @@ public function index() {
 
 		if ($this->defaultBoardService->checkFirstRun($this->userId, $this->appName)) {
 			if ($this->permissionService->canCreate()) {
-				$this->defaultBoardService->createDefaultBoard($this->l10n->t('Personal'), $this->userId, '000000');
+				$this->defaultBoardService->createDefaultBoard($this->l10n->t('Personal'), $this->userId, '317CCC');
```

Comment:
```suggestion
				$this->defaultBoardService->createDefaultBoard($this->l10n->t('Personal'), $this->userId, '0087C5');
```

Let's use the actual nextcloud color ;)